app/build.gradle.kts

```diff
@@ -130,6 +134,7 @@ dependencies {
     implementation(libs.compose.material.icons)
     implementation(libs.compose.material.ripple)
     implementation(libs.compose.ui.preview)
+    implementation(libs.compose.ui.viewbinding)
```

Comment:
Please Ignore. This is removed in part 2.
This was part of the prev spike work.

Reply:
removed